server/sonar-server/src/main/java/org/sonar/server/computation/step/ComputationSteps.java

```diff
@@ -47,6 +47,8 @@
       // Read report
       ParseReportStep.class,
 
+      QualityGateStep.class,
```

Comment:
Don't you think that ```Step``` implementations should be named with verbs like ```RetrieveQualityGateStep``` or nouns  like ```QualityGateLoaderStep``` ? Current name ```QualityGateStep" does not help to understand the responsibility.